src/auth/signup/CSS/signup.scss

```diff
@@ -0,0 +1,122 @@
+@import '../../../commons/assets/variables/font.scss';
+@import '../../../commons/assets/variables/colors.scss';
+
+.signup-container {
+  margin:10px 0 20px 0;
+
+  h1 {
+    color: $primary-color;
+    font-family: $lato-font;
+    font-weight: 100;
+    font-size: 24px;
+  }
+
+  .form {
+    min-height: 500px;
+    width: 544px;
+    border: #cccccc solid 1px;
+    text-align: center;
+    margin-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .heading {
+    width: 387px;
+    height: 30px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    opacity: 99%;
+  }
+
+  .form input, button {
+    width: 294px;
+    height: 38px;
+    margin: 20px;
+
+  }
+  .email {
+    margin-top: 30px;
+  }
+  .form button {
+    margin-top: 20px;
+    background-color: #00a4d3;
+    text-align: center;
+    color: white;
+    font-weight: 100;
+    font-family: $lato-font;
+  }
+
+  ::placeholder{
+    color: #000000;
+    font-family: "Source Sans Pro";
```

Comment:
consider using the font variable in the `commons/assets/variables/` folders everywhere in your style files where you are referring to font-family

Reply:
Resolved